doc/DevelopersGuide/README.md

```diff
@@ -0,0 +1,12 @@
+# CCPP Framework Developers Guide
+
+The PDF files in this directory are generated in the following manner:
+
+When the ``ccpp_prebuild.py`` script is run for the single column model (SCM) or
```

Comment:
SCM and the ufs-weather-model (not FV3) or any other host model  are all host models. The sentence reads as if SCM wasn't.